extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -281,33 +281,20 @@ export const TransactionDetails = ({ goBack }: { goBack: () => void }) => {
         Number(assetBalance.decimals),
       );
 
-      const sourceAccount = await sorobanServer.getAccount(publicKey);
-      const contract = new SorobanClient.Contract(assetAddress);
-      const contractOp = contract.call(
-        "transfer",
-        ...[
-          accountIdentifier(publicKey), // from
-          accountIdentifier(destination), // to
-          numberToI128(parsedAmount.toNumber()), // amount
-        ],
-      );
+      const params = [
+        new SorobanClient.Address(publicKey).toScVal(), // from
+        new SorobanClient.Address(destination).toScVal(), // to
+        SorobanClient.nativeToScVal(parsedAmount.toNumber(), { type: "i128" }), // amount
```

Comment:
`new SorobanClient.XdrLargeInt('i128', parsedAmount).toScVal()` may be faster/more direct, I think?

Reply:
done in 05fa8fe00e388f8cbf24ba50b6f75548e8ba489d